auto_nag/scripts/inactive_ni_pending.py

```diff
@@ -153,7 +153,6 @@ def get_action_type(bug, inactive_ni):
         if (
             len(bug["needinfo_flags"]) == 1
             and bug["type"] == "defect"
-            and bug["severity"] in ("--", "normal", "minor", "trivial")
```

Comment:
Nit: maybe we should make it explicit in case we change the conditions of the previous branch

Reply:
Changing the conditions in the previous branch will affect everything after anyway. I feel reducing the number of conditions will make it easier to follow.